packages/core/src/dialects/abstract/data-types.ts

```diff
@@ -2190,11 +2469,16 @@ export class GEOMETRY extends AbstractDataType<GeoJson> {
  * If this type is not supported, an error will be raised.
  *
  * @example <caption>Defining a Geography type attribute</caption>
```

Comment:
![image](https://user-images.githubusercontent.com/13023439/212564630-d8edc2c4-5fdf-46df-890b-d81a564ababe.png)
At this stage I think you get that this is an issue